.travis.yml

```diff
@@ -13,14 +13,10 @@ cache:
 os:
     - linux
 
-stage: Comprehensive tests
```

Comment:
Yes, I had noticed this in my own package too. Does mean we should update `package-template` as well!